modules/appeals_api/spec/fixtures/v2/valid_200996_extra.json

```diff
@@ -1,189 +1,189 @@
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "left knee",
-          "decisionDate": "1900-01-07",
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "right knee",
-          "decisionDate": "1900-01-08",
-          "disagreementArea": "Rating"
-        }
+        "email": "bob@bobbytablesemail.com",
+        "timezone": "America/Chicago"
       },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "left foot",
-          "decisionDate": "1900-01-09",
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "right foot",
-          "decisionDate": "1900-01-10",
-          "socDate": "2021-01-08",
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "left hand",
-          "decisionDate": "1900-01-11",
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "right hand",
-          "decisionDate": "1900-01-12",
-          "disagreementArea": "Rating"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "fever",
-          "decisionDate": "1900-01-13",
-          "disagreementArea": "Service connection"
-        }
-      },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "lupus",
-          "decisionDate": "1900-01-14",
-          "socDate": "2020-09-23"
-        }
+      "claimant": {
+        "address": {
+          "addressLine1": "456 First St",
+          "addressLine2": "Apt 5",
+          "addressLine3": "Box 1",
+          "city": "Detroit",
+          "countryCodeISO2": "US",
+          "stateCode": "MI",
+          "zipCode5": "48070"
+        },
+        "phone": {
+          "countryCode": "1",
+          "areaCode": "555",
+          "phoneNumber": "8111100",
+          "phoneNumberExt": "4"
+        },
+        "email": "someone@email.com",
+        "timezone": "America/Detroit"
       },
-      {
-        "type": "contestableIssue",
-        "attributes": {
-          "issue": "cootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescootiescooties",
-          "decisionDate": "1900-01-15",
-          "disagreementArea": "Service connectionService connectionService connectionService connectionService connection"
+      "socOptIn": true,
+      "informalConference": true,
+      "informalConferenceContact": "veteran",
+      "informalConferenceTime": "800-1200 ET",
+      "informalConferenceRep": {
+        "firstName": "Helen",
```

Comment:
The representative name looks pretty bad on the maxlength pdf. Let's put in a ticket to get that cleaned up.